test/local/routes/account_devices.js

```diff
@@ -0,0 +1,367 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var sinon = require('sinon')
+
+var test = require('tap').test
+var mocks = require('../../mocks')
+var getRoute = require('../../routes_helpers').getRoute
+var makeRoutes = require('../../routes_helpers').makeAccountRoutes
+
+var P = require('../../../lib/promise')
+var uuid = require('uuid')
+var crypto = require('crypto')
+
+var error = require('../../../lib/error')
+
+function runTest (route, request, assertions) {
```

Comment:
Looks like this is repeated verbatim in each of the new test modules, can we move it into the helpers?